src/info.rs

```diff
@@ -0,0 +1,76 @@
+// Copyright 2022 Adobe. All rights reserved.
+// This file is licensed to you under the Apache License,
+// Version 2.0 (http://www.apache.org/licenses/LICENSE-2.0)
+// or the MIT license (http://opensource.org/licenses/MIT),
+// at your option.
+// Unless required by applicable law or agreed to in writing,
+// this software is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR REPRESENTATIONS OF ANY KIND, either express or
+// implied. See the LICENSE-MIT and LICENSE-APACHE files for the
+// specific language governing permissions and limitations under
+// each license.
+
+use std::path::Path;
+
+use anyhow::Result;
+use c2pa::{IngredientOptions, ManifestStore};
+
+/// display additional C2PA information about the asset (not json formatted)
+pub fn info(path: &Path) -> Result<()> {
+    struct Options {}
+    impl IngredientOptions for Options {
+        fn thumbnail(&self, _path: &Path) -> Option<(String, Vec<u8>)> {
+            None
+        }
+    }
+    let ingredient = c2pa::Ingredient::from_file_with_options(path, &Options {})?;
+    println!("Information for {}", ingredient.title());
+    //println!("instanceID = {}", ingredient.instance_id());
+    if let Some(provenance) = ingredient.provenance() {
+        if !provenance.starts_with("self#jumbf=") {
+            println!("Cloud URL = {}", provenance);
+        } else {
+            println!("XMP provenance URI = {}", provenance);
+        }
+    }
+    if let Some(manifest_data) = ingredient.manifest_data() {
+        let file_size = std::fs::metadata(path).unwrap().len();
+        println!(
+            "C2PA manifest store size = {} ({:.2}% of {})",
+            manifest_data.len(),
+            file_size as f64 / manifest_data.len() as f64,
```

Comment:
I would think you would need a `* 100` here.

Reply:
yup this was wrong